app/dashboard/models.py

```diff
@@ -5026,17 +5029,39 @@ class Poll(SuperModel):
     hackathon = models.ManyToManyField(HackathonEvent)
 
 
+class PollMedia(SuperModel):
+    name = models.CharField(max_length=350)
+    image = models.ImageField(
```

Comment:
I have a feeling this won't work in prod because of the way static files are handled - @thelostone-mc @octavioamu what do y'all think?

Reply:
Hmm it's done like that for a few other image fields in our model 🤔